pkg/reckoner/plot.go

```diff
@@ -122,27 +122,20 @@ func (c Client) TemplateAll(templateOutputDir string) (fullOutput string, err er
 			color.Red(err.Error())
 		}
 
-		out, err := c.TemplateRelease(release.Name)
+		out, err := c.TemplateRelease(release.Name, templateOutputDir)
 		if err != nil {
 			color.Red(err.Error())
 			continue
 		}
 
-		if len(templateOutputDir) > 0 {
-			err = c.WriteSplitYaml([]byte(out), templateOutputDir, release.Name)
-			if err != nil {
-				color.Red(err.Error())
-				os.Exit(1)
-			}
-		}
-
 		fullOutput = fullOutput + out
 	}
+
 	return fullOutput, nil
 }
 
 // TemplateRelease does the same thing as TemplateAll but only for one release
-func (c Client) TemplateRelease(releaseName string) (string, error) {
+func (c Client) TemplateRelease(releaseName string, templateOutputDir string) (string, error) {
```

Comment:
I think for a future PR, it would be good to make templateOutputDir a field of the client struct, rather than passing it into these functions separately, since it is a global configuration passed in via the CLI. 

Non-blocking for this PR